rpa-builder/6.2/modules/ROOT/pages/_partials/toolbox-excel-controlled-loop.adoc

```diff
@@ -0,0 +1,7 @@
+//tag::troubleshootingSettings[]
+*Troubleshooting settings* 
+
+* *Disable recalculation of date formulas* 
++
+Disables the recalculation of cell values formatted as DateTime to prevent RPA Builder from becoming unresponsive until it finishes calculating formulas that contain many dependencies. 
```

Comment:
```suggestion
Disables the recalculation of cell values formatted as DateTime to prevent RPA Builder from becoming unresponsive while it finishes calculating formulas that contain many dependencies. 
```